### PR TITLE
port gbagfx and rsfont to lodepng

### PR DESCRIPTION
**Pull request by camthesaxman**
This removes the libpng and zlib dependencies from the tools.

### CODE REVIEW

File: tools/rsfont/convert_png.c [hunk 1]
```diff
@@ -1,169 +1,131 @@
...
-		row_pointers[i] = (png_bytep)(image->pixels + (i * rowbytes));
+    if (state.info_png.color.palettesize > 256)
+        FATAL_ERROR("Images with more than 256 colors are not supported.\n");
 
-	if (setjmp(png_jmpbuf(png_ptr)))
-		FATAL_ERROR("Error reading from \"%s\".\n", path);
+    palette->numColors = state.info_png.color.palettesize;
 
-	png_read_image(png_ptr, row_pointers);
+    // TODO: Why is LodePNG getting the palette size wrong?
```
Comment (sceptillion) [inline tools/rsfont/convert_png.c]: I don't know if it is getting it wrong. There appear to be some PNG files with more colors in their palette than they should have. For instance, "graphics/trade/gba.png" has 64 colors in its PLTE chunk even though it should have 16.

**Comment by camthesaxman** [comment]
Converting .4bpp, .8bpp, etc to PNG is broken, so I'll need to fix that before this gets merged.

**Comment by PikalaxALT** [comment]
Is this still alive?

**Comment by PikalaxALT** [comment]
PR is dead, closing.

**Pull request closed without merging**